src/mgr/DaemonServer.cc

```diff
@@ -711,7 +711,7 @@ class CommandContext {
     }
 
     if (r == 0) {
-      dout(4) << __func__ << " success" << dendl;
+      dout(20) << "success" << dendl;
```

Comment:
why do you want to remove `__func__` from the output?

Reply:
```
2019-07-25T11:12:18.433+0200 7f43f8d37700  4 mgr.server reply reply success
```
`dout` expands to something that already contains `__func__`

Reply:
sorry i didn't notice that:

https://github.com/ceph/ceph/blob/d5ed1eaa1ba48c0597903be66a526039e66eac48/src/mgr/DaemonServer.cc#L43